src/router.js

```diff
@@ -5,9 +5,24 @@ const router = createRouter({
   routes: [
     {
       path: "/",
+      name: "homepage",
+      component: () => import("./views/Homepage.vue"),
+    },
+    {
```

Comment:
Are we going to use the Login.vue anymore since you have NewLogin.vue? If not, let's delete the route here and the .vue file.